src/pages/explorePage/ExplorePage.tsx

```diff
@@ -99,6 +138,41 @@ const ExplorePageContent = ({
     hasPrereqs,
   };
 
+  const mapFilterStateToURL = (fs: SearchFilterState) => {
+    return {
+      exclude: fs.courseCodes
+        .map((bool, index) => (bool ? null : index))
+        .filter((index) => index !== null),
+      numCourseRatings: !fs.numCourseRatings ? null : fs.numCourseRatings,
```

Comment:
Can probably just do `fs.numCourseRatings || null` instead of ternary (and below), unless that linter rule is still active then nvm

Reply:
Good point, changed.